src/Symfony/Bundle/WebProfilerBundle/composer.json

```diff
@@ -24,13 +24,13 @@
         "twig/twig": "~1.34|~2.4"
     },
     "require-dev": {
-        "symfony/config": "~3.4|~4.0",
+        "symfony/config": "~4.2",
```

Comment:
As TwigBundle depends on it, and WebProfilerBundle depends on TwigBundle, this gets installed all the time already. I suggest making it a normal requirement instead of a conflict rule (this might even perform better in Composer, as a `<4.2` conflict rule matches lots of packages)